src/generators/mappersGenerator.ts

```diff
@@ -105,7 +106,9 @@ function writeMapper(writer: CodeBlockWriter, mapper: Mapper) {
   } = type as CompositeMapperType;
   const isReferenceDicriminator = isString(polymorphicDiscriminator);
   writer.block(() => {
-    writeObjectProps(restMapper, writer)
+    const { defaultValue, ...mapperProps } = restMapper;
+    writeDefaultValue(writer, defaultValue, restType);
```

Comment:
Default value is not specific to XML-based services. We were just not handling this before this change. The reason I started handling this here is because some types need some special stuff... For example you can have a ByteArray with a default value = "". However we cannot just generate the mapper with defaultValue: "", we need to convert "" to  `new Uint8Array("")`.